packages/plugin/src/theia.d.ts

```diff
@@ -6705,6 +6705,18 @@ export module '@theia/plugin' {
         /**
          * A base file path to which this pattern will be matched against relatively.
          */
+        baseUri: Uri;
+
+        /**
+         * A base file path to which this pattern will be matched against relatively.
```

Comment:
I know this is the text from VSCode, but the redundant prepositions alarm me:

```suggestion
         * A base file path against which this pattern will be matched relatively.
```